### PR TITLE
Replace the static components array with an api call to repo data to ensure we get latest array of origami components

### PR DESCRIPTION
**Pull request by JakeChampion**
This resolves https://github.com/Financial-Times/origami-component-converter/issues/21

### CODE REVIEW

File: __snapshots__/components.js [hunk 1]
```diff
@@ -1838,7 +1838,7 @@ exports['occ o-forms-7.0.0 o-forms-7.0.0 without CIRCLE_REPOSITORY_URL set match
...
+      "contents": "{\n\t\"browser\": \"browser.js\",\n\t\"eslintIgnore\": [\n\t\t\"browser.js\",\n\t\t\"dist/\"\n\t],\n\t\"keywords\": [\n\t\t\"origami\",\n\t\t\"component\",\n\t\t\"ft\"\n\t],\n\t\"name\": \"@financial-times/o-forms\",\n\t\"version\": \"0.0.0\",\n\t\"dependencies\": {\n\t\t\"o-brand\": {},\n\t\t\"o-colors\": {},\n\t\t\"o-grid\": {},\n\t\t\"o-icons\": {},\n\t\t\"o-loading\": {},\n\t\t\"o-normalise\": {},\n\t\t\"o-typography\": {}\n\t},\n\t\"component\": \"o-forms\",\n\t\"scripts\": {},\n\t\"devDependencies\": {}\n}\n"
```
Comment (JakeChampion) [inline __snapshots__/components.js:1841]: the dependencies objects are not prefixed with `@financial-times/`.

File: __snapshots__/components.js [hunk 2]
```diff
@@ -1838,7 +1838,7 @@ exports['occ o-forms-7.0.0 o-forms-7.0.0 without CIRCLE_REPOSITORY_URL set match
...
+      "contents": "{\n\t\"browser\": \"browser.js\",\n\t\"eslintIgnore\": [\n\t\t\"browser.js\",\n\t\t\"dist/\"\n\t],\n\t\"keywords\": [\n\t\t\"origami\",\n\t\t\"component\",\n\t\t\"ft\"\n\t],\n\t\"name\": \"@financial-times/o-forms\",\n\t\"version\": \"0.0.0\",\n\t\"dependencies\": {\n\t\t\"@financial-times/o-brand\": {},\n\t\t\"@financial-times/o-colors\": {},\n\t\t\"@financial-times/o-grid\": {},\n\t\t\"@financial-times/o-icons\": {},\n\t\t\"@financial-times/o-loading\": {},\n\t\t\"@financial-times/o-normalise\": {},\n\t\t\"@financial-times/o-typography\": {}\n\t},\n\t\"component\": \"o-forms\",\n\t\"scripts\": {},\n\t\"devDependencies\": {}\n}\n"
```
Comment (JakeChampion) [inline __snapshots__/components.js:1841]: The dependencies seem to have `{}` instead of versions...

File: lib/npm.js [hunk 3]
```diff
@@ -5,8 +5,16 @@ let {entries: getEntries, keys: getKeys} = Object
...
 
+const origamiComponentNames = (async function() {
+	const repoData = new RepoDataClient()
+	const components = await repoData.listRepos({
+		type: 'module'
+	})
+	const componentNames = components.map(component => component.name)
+	return componentNames
+}())
 /**
```
Comment (chee) [inline lib/npm.js:18]: i really like this ifee memo style, i've seen you do it in a few places

File: lib/npm.js [hunk 4]
```diff
@@ -162,7 +170,7 @@ export let createDependencyVersion = async ([name, version]) => {
  */
 export let createDependency = async ([name, version]) => {
 	let npmName = await createDependencyName(name)
-	let npmVersion = await createDependencyVersion([name, version])
+	let npmVersion = createDependencyVersion([name, version])
```
Comment (chee) [inline lib/npm.js:173]: is this missing?
```suggestion
	let npmVersion = await createDependencyVersion([name, version])
```

Comment (JakeChampion) [reply lib/npm.js]: This was missing yeah, good catch. I removed it as part of a warning I got in my editor about `createDependencyVersion` not needing to be an async function as it doesn't use await. I changed `createDependencyVersion` to be a normal function and removed this await call but it made the tests fail. When I went to revert my work I forgot to add this `await` back 🙈

File: lib/npm.js (hunk 3 above)
Comment (JakeChampion) [reply lib/npm.js:18]: I think I like it to :-)

**Pull request merged**